library/init/meta/interactive.lean

```diff
@@ -1636,11 +1636,12 @@ do gs ← get_goals,
 /--
 The tactic `specialize h a₁ ... aₙ` works on local hypothesis `h`. The premises of this hypothesis, either universal quantifications or non-dependent implications, are instantiated by concrete terms coming either from arguments `a₁` ... `aₙ`. The tactic adds a new hypothesis with the same name `h := h a₁ ... aₙ` and tries to clear the previous one.
 -/
-meta def specialize (p : parse texpr) : tactic unit :=
+meta def specialize' (p : parse texpr) : tactic unit :=
```

Comment:
Ah, you probably don't want to change the name here!
```suggestion
meta def specialize (p : parse texpr) : tactic unit :=
```

Reply:
ah yes, I've copied this from the PR. The prime shouldn't be there.